src/Template/Element/advanced_search_form.ctp

```diff
@@ -29,6 +29,21 @@ echo $this->Form->create('AdvancedSearch', [
 ]);
 ?>
 
+<?php if ($this->request->getParam("?")): ?>
+    <md-divider>
+        <md-button class="md-primary md-raised"
+                    href="<?= $this->Url->build([
+                        "controller" => "Sentences",
+                        "action" => "advanced_search",
+                        "?" => [],
+                    ]);?>">
+            <?= __('Clear') ?>
+        </md-button>
```

Comment:
I would have put the button *after* the text. I think it makes it clearer what the button is clearing. You could as well make the button say "Clear form" for more clarity.

Reply:
That does read more naturally. I thought that having a big green button just floating in the middle of the page looked a little odd, so I tried right aligning it. Compare 9c1f780 and d99b11f.

Reply:
Having it on the right looks odder to me than having a floating button. Maybe you can lower the attractiveness of the button by removing `md-primary` and/or `md-raised` and/or replacing it by a text link.